monkey/monkey_island/cc/app.py

```diff
@@ -94,7 +95,10 @@ def setup_authentication(app, data_dir):
     app.config["SECURITY_USERNAME_REQUIRED"] = True
     app.config["SECURITY_REGISTERABLE"] = True
     app.config["SECURITY_SEND_REGISTER_EMAIL"] = False
+
+    app.config["SECURITY_TOKEN_MAX_AGE"] = AUTH_EXPIRATION_TIME
     # Ignore CSRF, because it's irrelevant for javascript applications
+    app.config["WTF_CSRF_ENABLED"] = False
```

Comment:
Is this relevant? There should be no way for users to retrieve the form...

Reply:
I think it is if we want to disable CSRF.